dev/io.openliberty.microprofile.health.3.0.internal/src/io/openliberty/microprofile/health30/internal/HealthCheck30HttpResponseBuilder.java

```diff
@@ -23,19 +23,21 @@
 
 import java.io.IOException;
 import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.Iterator;
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
 
+import javax.json.Json;
```

Comment:
I totally messed up which file to update here.  It was meant to be on the health.3.1.internal project needing to be updated like the 3.0 one.

Reply:
In the 3.1 one we may not even need jsonp dependency I guess since it isn't used in the 3.1 internal bundle.  Only need to remove the jackson dependency and import.

Reply:
ok, I think the bnd file is right now? maybe?